PayPal.xcodeproj/xcshareddata/xcschemes/Card.xcscheme

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Under `xcshareddata/xcschemes`, on main we have a scheme for `PaymentsCore` and `PayPal`, but not `Card`. This was automatically generated (maybe when I ran the `Card` scheme?). Not sure how it escaped source control til now.